### DISCUSSION TITLE
Do I really need a billed subscription when using it with GPT3.5 ?

### DISCUSSION THREAD

**Discussion by JannikSchneider12** [melih-unsal/DemoGPT#31] [General]
Hello,

I wanted to test DemoGPT, but always getting this error: "Error occurred when generating search queries: You exceeded your current quota, please check your plan and billing details".

Do I really need to check the payment plan (and pay for every request) or is there another way?

Thanks for your work and help

**Comment by melih-unsal** [comment]
Hi @JannikSchneider12 ,

Thank you for reaching out and for your interest in DemoGPT. I'm sorry for the inconvenience you're experiencing.

Yes, as of now, the OpenAI API does require a payment method as it is currently set up on a pay-per-use basis. The error message you're seeing is related to this payment structure and indicates that the number of requests you're making has exceeded the quota for your current plan.

However, I'm pleased to share that for the upcoming releases, we're working on introducing Llama 2 as an option. With Llama 2, you will not need to worry about payment methods or exceeding quotas. This is because Llama 2 will be completely runnable locally, eliminating the need for an API call and its associated cost.

We'll make sure to announce when this feature is live. In the meantime, if you have any further questions or concerns, feel free to let us know. Your feedback is invaluable in helping us improve our services.

Thank you again for your patience and support!

Best regards,
Melih.
